### PR TITLE
Readme update.

### PR DESCRIPTION
**Pull request by chrisbrickhouse**
I've updated the installation instructions in the Readme based on the steps I needed to get a local version running on my linux machine. I also cleaned up the formatting a bit.

### CODE REVIEW

**Comment by jlwhelan28** [comment]
Hello, new face here. Incoming JS noob question... And likely an XY problem y'all can ignore. 

@chrisbrickhouse Commenting out `forcessl.mw.js` was also the only way I could get a localhost up and running without more knowledge of JS. This function seems to simply redirect `http://` to `https://`, but interferes when running locally somehow... 

I thought the `https` issue was not having a certificate for localhost, but after generating a certificate with openSSL, and creating the Express server as below, I was able to get past the secure connection error, but still can't connect to localhost properly. Then after omitting `forcessl.mw.js`, I can connect to localhost through both `http` and `https`, with either the below server configuration or the original use of `Express`

Basically what I'm trying to understand is why does commenting out the above require work?

```JavaScript
const PORT = process.env.PORT || 8000;

https.createServer({
	key:fs.readFileSync('server.key'),
	cert: fs.readFileSync('server.cert'),
}, app)
.listen(PORT, function () {
	console.log(`Homebrewery listening on port:${PORT}`)
})

// app.listen(PORT);
// console.log(`server on port:${PORT}`);
```

**Comment by datdamnzotz** [comment]
Might want to just update the readme to say something like

Add an environment variable like this:
`NODE_ENV=local` 
bypasses the force SSL.

As seen here in the code https://github.com/naturalcrit/homebrewery/blob/master/server/forcessl.mw.js#L2

**Comment by tcyrus** [comment]
@calculuschild Sorry for bothering you, but I'm curious on the status of this PR

**Comment by calculuschild** [comment]
I have accepted this PR. Changed the text to encourage using environment variables rather than commenting out lines of code.
